tsconfig.json

```diff
@@ -1,14 +1,14 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "esnext",
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "noEmit": true,
     "esModuleInterop": true,
-    "module": "esnext",
+    "module": "commonjs",
```

Comment:
you cannot parse the entire project as if it were nodejs, as the rest runs on the browser.

If you need to run code on the server, provide a `tsconfig.json` with `"module": "commonjs"` in the same folder as your server files, keeping the root `tsconfig.json` its original value

Reply:
module is still `commonjs` here